web/themes/new_weather_theme/templates/block/block--weathergov-current-conditions.html.twig

```diff
@@ -14,15 +14,13 @@
       {# First row is temperature #}
       <div class="display-flex">
         <div class="current-conditions-temp display-flex flex-row flex-align-start margin-right-2 text-primary-darker">
-          {{ content.temperature }}<span aria-hidden="true" class="font-body-xs display-inline-block left-full margin-top-05 margin-left-neg-2px">&deg;F</span>
-          <span class="usa-sr-only">℉</span>
+          {{ content.temperature }}<span class="font-body-xs display-inline-block left-full margin-top-05 margin-left-neg-2px">&deg;F</span>
         </div>
         <div class="margin-top-05 position-relative">
           <div class="font-mono-2xs font-family-mono text-base text-uppercase">{{ "Feels like" | t }}</div>
           <div class="text-primary-darker">
-            <p class="margin-top-2px font-body-md">{{ content.feels_like }}
-              <span aria-hidden="true" class="font-body-3xs position-absolute margin-top">&deg;F</span>
-              <span class="usa-sr-only">℉</span>
+            <p class="margin-top-2px font-body-md">
+              {{ content.feels_like }}<span class="font-body-3xs position-absolute margin-top">&deg;F</span>
```

Comment:
We don't need screen reader-specific handling here because these entire blocks are hidden from screen readers.